# Portuguese translations for Hue.
# Copyright (C) 2012 Cloudera, Inc
# This file is distributed under the same license as the Hue project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2012.
#
msgid ""
msgstr ""
"Project-Id-Version: Hue VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2018-10-16 20:02-0700\n"
"PO-Revision-Date: 2012-11-07 13:06-0800\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language: pt\n"
"Language-Team: pt <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.5.1\n"

#: src/hadoop/conf.py:66
msgid "NameNode logical name."
msgstr "Nome lógico NameNode."

#: src/hadoop/conf.py:108
msgid "JobTracker logical name."
msgstr "Nome lógico JobTracker."

#: src/hadoop/conf.py:137
msgid "Resource Manager logical name."
msgstr "Nome lógico do Resource Manager."

#: src/hadoop/fs/hadoopfs.py:212
#, python-format
msgid "%(remote_dst)s already exists. Skipping."
msgstr "%(remote_dst)s já existente. A ignorar."

#: src/hadoop/fs/hadoopfs.py:215
#, python-format
msgid "%(remote_dst)s does not exist. Trying to copy."
msgstr "%(remote_dst)s inexistente. A tentar copiar."

#: src/hadoop/fs/hadoopfs.py:225
#, python-format
msgid "Copied %s -> %s."
msgstr "Copiado %s -> %s."

#: src/hadoop/fs/hadoopfs.py:227
#, python-format
msgid "Copying %s -> %s failed."
msgstr "Falha ao copiar %s -> %s."

#: src/hadoop/fs/hadoopfs.py:232
#, python-format
msgid "Skipping %s (not a file)."
msgstr "A ignorar %s (não é um ficheiro)."

#: src/hadoop/fs/hadoopfs.py:268 src/hadoop/fs/hadoopfs.py:271
#: src/hadoop/fs/hadoopfs.py:274 src/hadoop/fs/hadoopfs.py:277
#: src/hadoop/fs/hadoopfs.py:280 src/hadoop/fs/hadoopfs.py:283
#: src/hadoop/fs/hadoopfs.py:286
#, python-format
msgid "%(function)s has not been implemented."
msgstr "%(function)s não foi implementada."

#: src/hadoop/fs/upload.py:68
msgid "No HDFS found"
msgstr ""

#: src/hadoop/fs/upload.py:79
#, python-format
msgid "User %s does not have permissions to write to path \"%s\"."
msgstr ""

#: src/hadoop/fs/webhdfs.py:298 src/hadoop/fs/webhdfs.py:335
#, python-format
msgid "File %s not found"
msgstr "Ficheiro %s não encontrado"

#: src/hadoop/fs/webhdfs.py:338
#, python-format
msgid "File %s is a directory"
msgstr "O ficheiro %s é um directório"

#: src/hadoop/fs/webhdfs.py:342
#, python-format
msgid "File %s is already trashed"
msgstr "O ficheiro %s já está no lixo"

#: src/hadoop/fs/webhdfs.py:370
#, python-format
msgid "Delete failed: %s"
msgstr "Falha ao eliminar: %s"

#: src/hadoop/fs/webhdfs.py:399
#, python-format
msgid "File %s is not in trash"
msgstr "O ficheiro %s não está no lixo"

#: src/hadoop/fs/webhdfs.py:413
#, python-format
msgid "Path %s already exists."
msgstr "O caminho %s já existe."

#: src/hadoop/fs/webhdfs.py:441
#, python-format
msgid "Mkdir failed: %s"
msgstr "O Mkdir falhou: %s"

#: src/hadoop/fs/webhdfs.py:456
#, python-format
msgid "Rename failed: %s -> %s"
msgstr "Falha ao alterar nome: %s -> %s"

#: src/hadoop/fs/webhdfs.py:462 src/hadoop/fs/webhdfs.py:466
#, python-format
msgid "'%s' is not a directory"
msgstr "%s não é um directório"

#: src/hadoop/fs/webhdfs.py:675
#, python-format
msgid "Copy src '%s' does not exist"
msgstr "A origem da cópia \"%s\" não existe"

#: src/hadoop/fs/webhdfs.py:677
#, python-format
msgid "Copy src '%s' is a directory"
msgstr "A origem da cópia \"%s\" é um directório"

#: src/hadoop/fs/webhdfs.py:679
#, python-format
msgid "Copy dst '%s' is a directory"
msgstr "O destino da cópia \"%s\" é um directório"

#: src/hadoop/fs/webhdfs.py:760
#, python-format
msgid "File not found: %s"
msgstr "Ficheiro não encontrado: %s"

#: src/hadoop/fs/webhdfs.py:776
#, python-format
msgid "Destination file %s exists and is not a directory."
msgstr "O ficheiro de destino %s já existe e não é um directório."

#: src/hadoop/fs/webhdfs.py:818
#, python-format
msgid "Failed to create '%s'. HDFS did not return a redirect"
msgstr "Falha ao criar \"%s\". O HDFS não devolveu um redireccionamento"

#: src/hadoop/fs/webhdfs.py:886
msgid "Failed to upload file. WebHdfs requires a valid username to upload files."
msgstr ""

#: src/hadoop/fs/webhdfs.py:913
#, python-format
msgid "Is a directory: '%s'"
msgstr "É um directório: '%s'"

#: src/hadoop/fs/webhdfs.py:931
msgid "Invalid argument to seek for whence"
msgstr "Argumento inválido para procurar por origem"

#: src/hadoop/fs/webhdfs.py:951
msgid "File not open for writing"
msgstr "O ficheiro não está aberto para escrita"

#: src/hadoop/fs/webhdfs.py:997
#, python-format
msgid "Filesystem root '/' should be owned by '%s'"
msgstr ""

#: src/hadoop/fs/webhdfs.py:1000
msgid "Failed to access filesystem root"
msgstr "Não foi possível aceder à raiz do sistema de ficheiros"

#: src/hadoop/fs/webhdfs.py:1008
#, python-format
msgid "Failed to create temporary file \"%s\""
msgstr "Falha ao criar o ficheiro temporário \"%s\""

#: src/hadoop/fs/webhdfs.py:1024
#, python-format
msgid "Failed to remove temporary file \"%s\""
msgstr "Falha ao remover o ficheiro temporário \"%s\""

#: src/hadoop/yarn/history_server_api.py:46
msgid "YARN cluster is not available."
msgstr ""

#: src/hadoop/yarn/mapreduce_api.py:47
#: src/hadoop/yarn/resource_manager_api.py:52
msgid "No Resource Manager are available."
msgstr "Nenhum Resource Manager disponível."

#: src/hadoop/yarn/resource_manager_api.py:143
msgid "YARN did not return any token field."
msgstr ""

#: src/hadoop/yarn/resource_manager_api.py:170
#, python-format
msgid "YARN RM returned a failed response: %s"
msgstr ""

#: src/hadoop/yarn/spark_history_server_api.py:53
msgid "No Spark History Server is available."
msgstr ""

#~ msgid "Filesystem root '/' should be owned by 'hdfs'"
#~ msgstr ""
#~ "A raiz do sistema de ficheiros "
#~ "\"/\" não deve ser propriedade de "
#~ "\"hdsf\""

